packages/template-retail-react-app/app/components/store-locator-modal/store-locator-content.jsx

```diff
@@ -5,21 +5,133 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import React from 'react'
+import React, {useState, useContext} from 'react'
 import {useIntl} from 'react-intl'
-import PropTypes from 'prop-types'
+
+// Components
 import {
     Heading,
     Accordion,
     AccordionItem,
-    Box
+    Box,
+    Button
 } from '@salesforce/retail-react-app/app/components/shared/ui'
 import StoresList from '@salesforce/retail-react-app/app/components/store-locator-modal/stores-list'
 import StoreLocatorInput from '@salesforce/retail-react-app/app/components/store-locator-modal/store-locator-input'
-import {STORE_LOCATOR_DISTANCE} from '@salesforce/retail-react-app/app/constants'
 
-const StoreLocatorContent = ({form, submitForm, storesInfo, searchStoresParams}) => {
+// Others
+import {
+    SUPPORTED_STORE_LOCATOR_COUNTRIES,
+    DEFAULT_STORE_LOCATOR_COUNTRY,
+    STORE_LOCATOR_DISTANCE,
+    STORE_LOCATOR_NUM_STORES_PER_LOAD,
+    STORE_LOCATOR_DISTANCE_UNIT
+} from '@salesforce/retail-react-app/app/constants'
+const NUM_STORES_PER_REQUEST_API_MAX = 200
```

Comment:
Observation: this constant seems to be placed at a different location than other constants. Is this intentional?

Reply:
Yes this is a maximum set by the API and shouldn't be configurable. So far we've put configurable constants for the store locator in `app/constants.js` and I thought this would be best to be here instead.

Reply:
good to know! If this shouldn't be configurable, let's add a comment so others in the future don't touch it